AWSS3/AWSS3TransferUtility.h

```diff
@@ -16,6 +16,8 @@
 #import <UIKit/UIKit.h>
 #import <AWSCore/AWSCore.h>
 
+FOUNDATION_EXPORT NSString *const AWSS3SessionInvalidatedNotification;
```

Comment:
How about `AWSS3TransferUtilityURLSessionDidBecomeInvalidNotification`? It's pretty verbose but more descriptive and following the delegate method name.